styleguide/writing-style.md

```diff
@@ -0,0 +1,183 @@
+More info on writing about pronouns and in both the [Google developer documentation style guide](https://developers.google.com/style/pronouns#gender-neutral-pronouns) and [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/grammar/nouns-pronouns#pronouns-and-gender) on pronouns.
+
+## Style and formatting
+
+This section covers the matters that go beyond language and provides guidelines for consistency in writing with a unified look.
+
+### Consistency
+
+Keep your writing consistent with itself and other docs. This means abbreviations, capitalization, hyphens, names of UI elements, etc.
+
+### Headings
+
+Use sentence case in all headings:
+
+_"The quick brown fox jumps over the lazy dog."_
+
+The standard case used in English prose. That is, only the first word is capitalized, except for proper nouns and other words which are generally capitalized by a more specific rule.
+
+Do not create multi-sentence headings, or add full stops/periods or other punctuation at the end of a heading.
+
+Refer to [Headings in the Markdown syntax guide](markdown-syntax-style.md#headings) for details on how to implement headings in Markdown.
+
+### Product Names (and product features)
+
+Only use Title Case (initial capital letters) for the name of the product (product names), and match capital letter usage in product names as an organization would use them in these products - for example:
+
+* 'Docker Compose overview' and not 'Docker Compose Overview' 
+* 'GitHub organization' and not 'GitHub Organization'
+
+> [!NOTE]
+> Avoid using Title Case for product features, that is, when mentioning or describing them in documentation.
+
+### UI elements
+
+UI elements should be formatted in italics. For example:
+
+To get your agent token, navigate to _Agents_, then select _Reveal Agent Token_.
+
+> [!NOTE]
+> Match the capitalization used in the Buildkite interface, even if title (or any other) case has been used for product features.
+
+Refer to [UI elements in the Markdown syntax guide](markdown-syntax-style.md#ui-elements) for details on how to write and present UI elements in the Buildkite docs.
+
+### Lists (bullet lists and numbered steps)
+
+Capitalize the first word; no full stops at the end if it's not a full sentence. If it's a full sentence, give it a full stop.
+
+See also what [Google](https://developers.google.com/style/lists#capitalization-and-end-punctuation) and [Microsoft](https://learn.microsoft.com/en-us/style-guide/scannable-content/lists#punctuation) say about lists.
+
+### Writing numbers
+
+Write out numbers up to 10, then use digits - '58 bugs in this script and just two hours to fix them all!'
+Long numbers use commas to separate thousands - '100,000,000.00'. When in doubt, see what [Google](https://developers.google.com/style/numbers) and [Microsoft](https://docs.microsoft.com/en-us/style-guide/numbers) say about numbers.
+
+Do not use numbers in page headings. Only use numbers less than 10 in section headings.
```

Comment:
I wonder if we want to soften this:

```suggestion
Avoid using numbers in page headings. Only use numbers less than 10 in section headings.
```

I can see cases where we might want to do that. For example, a multi-part tutorial.